main.css

```diff
@@ -138,3 +138,26 @@ main h1 {
 		float: right;
 	}
 }
+
+#sorted-list-op{
+	white-space: pre-line;
+	white-space: pre-wrap;
+	border: none;
+	width: 100%;
+	height: 100%;
+	padding: none;
+	margin: none;
+	outline: none;
+	overflow: hidden;
+	overflow-x: hidden;
```

Comment:
You don't need to set overflow-x: hidden if you've already set overflow: hidden

Reply:
I'm so sorry I forgot to clean the code before pushing! Will do it right away!

Reply:
On it! Completely overlooked that one!